interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -87,6 +88,16 @@
                 aoData.push({"name": "layout_id", "value": <?php echo js_escape($layout_id); ?>});
                 <?php } ?>
             },
+            fnDrawCallback: function(obj) {
+                const infoBox = $(".dataTables_wrapper p.moreResults");
+                if (infoBox.length) {
+                    infoBox.parent().remove();
+                }
+                if (obj.json && obj.json.iTotalHasMoreRecords) {
+                    $(".dataTables_wrapper .dataTables_info").parent().parent()
+                        .prepend("<div class='col-sm-12'><p class='alert alert-warning moreResults'>" + <?php echo xlj("Maximum displayable results reached.  Narrow your search"); ?> + "</p></div>");
```

Comment:
Avoid double spacing in translation constants.

Reply:
Fixed.